src/pocketmine/MemoryManager.php

```diff
@@ -32,16 +32,16 @@ class MemoryManager{
 	private $chunkCollect;
 	private $chunkTrigger;
 
-	private $chunkCache;
-	private $cacheTrigger;
-
 	/** @var \WeakRef[] */
 	private $leakWatch = [];
 
 	private $leakInfo = [];
 
 	private $leakSeed = 0;
 
+	private $chunkCache;
+	private $cacheTrigger;
+
```

Comment:
I did a commit to remove the weakref stuff here and then reverted it the wrong way